### DISCUSSION TITLE
How can we get aspects applied to a type?

### DISCUSSION THREAD

**Discussion by YBAZAN** [metalama/Metalama#78] [Q&A] [answered]
I migrated to the most recent version of Metalama (0.5.76-preview), and this code doesn't work anymore:

`namedType.Aspects<TAspect>()`

How can I get the aspects applies to a INamedType ?

**Comment by YBAZAN** [comment]
It seems this breaking change has been introduced at the 0.5.71-preview version

**Comment by gfraiteur** [comment, marked as answer]
It is now `namedType.Enhancements().GetAspects<.>()`.

**Reply by YBAZAN** [reply]
Thank you!
